### ISSUE TITLE
[1.9.5_2] Task was destroyed, but it is pending!

### ISSUE THREAD

**Issue by Moonshaft** [Just-Some-Bots/MusicBot#482]
This problem literally came out of nowhere and I have no idea how to fix it. It used to be that if the bot idled for a while it was just kind of stall out and need to be restarted, but now after a random amount of time it'll turn completely unresponsive to anyone and can't be closed via the traditional !shutdown. The batch file will have to be closed manually. Here's the error that I keep encountering when I notice that it's gone completely unresponsive to everyone (sometimes it is responsive and a song can be queue'd, but the bot doesn't post in its designated channel).

![bot](https://cloud.githubusercontent.com/assets/20808506/17354166/92c97d0c-590b-11e6-9855-24d9ca19fcde.png)

I've tried running:
pip3.5 install --upgrade -r requirements.txt in Git within the MusicBot folder
and
python -m pip install --upgrade pip in command prompt

And I've also tried uninstalling/reinstalling Python x64 but I haven't had any luck getting back to normal. I made some changes to the roles that can access things in the permissions.ini, but I even restored the folder back to the state that it was a week ago before the changes were made and I have the same problem. It's gotten to the point that it will be stuck at "Connecting..." when opened again and I'll need to restart it 5-7 time in order for it to even connect at all.
All help is appreciated.

**Comment by Moonshaft** [comment]
Ideas?

**Comment by Moonshaft** [comment]
Bump...

**Comment by Moonshaft** [comment]
Roughly a week or so I posted a problem that I was having with the music bot where for some reason it would completely stop receiving input at all whatsoever and would need to be restarted in order to accept input again. Here is an image link that shows when "task was destroyed but it is pending!" first pops up after 30-45 seconds of idle time but we're able to queue up another few songs without issue. When we let the pop idle out again for roughly a minute it completely lost all input response again and only after being restarted did it spam more "task was destroyed but it is pending" before getting stuck infinitely on "Connecting..." (runbot.bat needs to be completely restarted 1-5 times before it can connect again successfully): https://s20.postimg.org/e382375fx/bot4.png
I've tried completely uninstalling and reinstalling Discord, Git, Python, and the music bot itself with no luck. I've also made sure all of the dependancies were completely up to date so there's nothing outdated floating around. i've even deleted the folders that the messages are referencing and had them redownloaded when running runbot.bat but it hasn't made a difference. My only guess at this point is maybe there's some sort of hiccup going on that's causing the bot to be unable to "wake up" after idling which in turn causes it to be unresponsive until restarted.
I'm not really sure at this point what to look at and what to do so I'm looking for any suggestions at all. Many thanks to Jayden previously for taking a look and I apologize for pinging the dev but I feel stuck and frustrated @Imayhaveborkedit
After note: Forgot to mention. I noticed that after it finished playing "Everything is Alright" from the Motion City Soundtrack, the bot seemed to be stuck open even though the song was finished, kind of like how sometimes a mic is stuck open and giving input for no reason when it shouldn't on PTT and you need to disconnect/reconnect in order for it to go back to normal. You can see the slew of commands that it wasn't responding to during that "stuck open" phase before the disconnect. It wasn't responsive after I summoned it back and only after the restart did it spam

**Comment by K00sKlust** [comment]
Hi, I also had an idling issue with a script. The bot became unresponsible until I clicked the window and pressed enter.
Please read this [stackoverflow issue](http://stackoverflow.com/questions/33883530/why-is-my-command-prompt-freezing-on-windows-10). Maybe it is the solution for your problem.

**Comment by amit177** [comment]
If this is still an issue in the latest version, please open a new issue.
